woocommerce/woocommerce-gutenberg-products-block
# Update to node 16/npm 8

This PR updates the repo to use the current LTS of node and syncs .nvmrc and `package.json` engines to the same version (v16.13.0). I've moved away from `lts/*` in order to avoid automatically switching major versions. I think it's good practice to do these upgrades manually and cautiously but if anyone feels the opposite let's discuss :)

I've tested with install, build, run and test scripts and everything seems fine. The only problem I can foresee is releasing which I don't know too much about yet, so thoughts around that would be appreciated.

### Manual Testing

How to test the changes in this Pull Request:

1. Check that all the Github actions are green
2. Pull this branch to your local machine
3. `nvm install` or `nvm use` should switch you to use version ^16.13.0 and npm version ^v8.0.0
3. `npm run build` should build successfully
4. `npm run start` should run successfully
5. `npm run test` should run successfully

### Changelog

> Updated to node version ^16.13.0 and npm version ^v8.0.0
